src/custom-element.js

```diff
@@ -1897,14 +1897,15 @@ function createBaseCustomElementClass(win, elementConnectedCallback) {
     }
 
     /**
-     * Returns the original nodes of the custom element without any service
-     * nodes that could have been added for markup. These nodes can include
-     * Text, Comment and other child nodes.
-     * @return {!Array<!Node>}
+     * See dom.getRealChildNodes for the full description.
+     * TODO: migrate to the fn in dom.js.
+     * @deprecated
      * @package @final
+     *
+     * @return {!Array<!Node>}
      */
     getRealChildNodes() {
-      return query.childNodes(this, (node) => !isInternalOrServiceNode(node));
+      return dom.getRealChildNodes(this);
```

Comment:
Is the goal of this pattern to eventually remove `customElement.getRealChildNodes()` and replace with `getRealChildNodes(customElement)`?

Reply:
Resolved offline